pkg/manifests/config.go

```diff
@@ -186,7 +189,15 @@ func (cfg *TelemeterClientConfig) IsEnabled() bool {
 	return true
 }
 
-func NewConfig(content io.Reader) (*Config, error) {
+func (cps CollectionProfiles) String() string {
+	jointProfiles := make([]string, 0, len(cps))
+	for _, cp := range cps {
+		jointProfiles = append(jointProfiles, string(cp))
+	}
+	return strings.Join(jointProfiles, ", ")
```

Comment:
nit: `string.Builder` is be a better choice than creating a slice and joining.

Reply:
I love these nit's they help me write better code! Thank you!